### ISSUE TITLE
Question about how to get motion lib dataset

### ISSUE THREAD

**Issue by openhe-hub** [song-siqi/retarget2humanoid#1]
I am also discovering how to do smpl retargeting. After viewing your code, may I ask how to get retarget result to motion lib dataset like boxing.npy? Because after retargeting, I still dont have data like root_vel... Thank you.

**Comment by openhe-hub** [comment]
Oh use this can be solved: 
# NOTE: another version of output which includes more information
    (
        root_pos,
        root_rot,
        dof_pos,
        root_vel,
        root_ang_vel,
        dof_vel,
        keypoint_trans
    ) = retarget.forward_trackable(
        joint_pos=joint_pos.detach(),
        root_trans=root_trans.detach(),
        root_ori=root_ori.detach()
    )

    output = {
        'file_name': file_name,
        'fps': retarget.data.frame_rate,
        'time_length': retarget.data.time_length,
        'num_frames': retarget.data.num_frames,
        'root_pos': root_pos,
        'root_rot': root_rot,
        'dof_pos': dof_pos,
        'root_vel': root_vel,
        'root_ang_vel': root_ang_vel,
        'dof_vel': dof_vel,
        'keypoint_trans': keypoint_trans
    }
